src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -59,7 +59,8 @@ public void Initialize(ExtensionConfigContext context)
                 var binding = context.AddBindingRule<QueueAttribute>();
                 binding
                     .AddConverter<byte[], IStorageQueueMessage>(ConvertByteArrayToCloudQueueMessage)
-                    .AddConverter<string, IStorageQueueMessage>(ConvertStringToCloudQueueMessage);
+                    .AddConverter<string, IStorageQueueMessage>(ConvertStringToCloudQueueMessage)
+                    .AddOpenConverter<OpenType.Poco, IStorageQueueMessage>(ConvertPocoToCloudQueueMessage);
```

Comment:
> .AddOpenConverter<OpenType.Poco, IStorageQueueMessage>(ConvertPocoToCloudQueueMessage); [](start = 19, length = 88)

This is formalizing our "Poco --> Queue" rule. Before, it was kind of a hacky representation. This also means other extensions can implement the same rule using the same definition of "Poco" that Queue uses.